react/admin/OrganizationDetails/OrganizationDetailsCollections.tsx

```diff
@@ -42,17 +42,19 @@ const OrganizationDetailsCollections = ({
   const {
     data: collectionsData,
     refetch: refetchCollections,
-  } = useQuery(GET_COLLECTIONS, { ssr: false })
+    loading,
+  } = useQuery(GET_COLLECTIONS, {
+    variables: collectionPaginationState,
+    notifyOnNetworkStatusChange: true,
+    ssr: false,
+  })
 
   /**
    * Effects
    */
 
   useEffect(() => {
-    if (
-      !collectionsData?.collections?.items?.length ||
-      collectionOptions.length
-    ) {
+    if (!collectionsData?.collections?.items?.length) {
```

Comment:
**question:** could you explain why this line was changed? Why the check on `collectionOptions.lenght` was removed?

Reply:
The effect of this check is that it's impossible to advance to the next page of available collections. The code sees that `collectionOptions` already has length (the first page of results) and therefore skips setting the second page of results to state.